pyopencl/array.py

```diff
@@ -42,6 +42,9 @@
         get_common_dtype as _get_common_dtype_base)
 from pyopencl.characterize import has_double_support
 from pyopencl import cltypes
+from numbers import Number
+
+SCALAR_CLASSES = (Number, np.number, np.bool_, bool)
```

Comment:
Doesn't `isinstance(x, np.number)` imply `isinstance(x, Number)`?

Reply:
Yep removed the redundancy here. Thanks! 21fa507